theme.xml

```diff
@@ -125,6 +125,16 @@ https://github.com/anthonycaccese/art-book-next-es
    <include>./colors.xml</include>
    <include ifSubset="color-scheme:custom">${themeCustomizationsPath}colors.xml</include>
 
+   <!--
```

Comment:
change this section to this:
```
   <!--
   ///
   Subset: Splash Screen
   ///
   -->
   <subset name="splash-screen" displayName="Splash Screen">
      <include name="default" displayName="Default" />
      <include name="custom" displayName="Custom" />
   </subset>
```

Reply:
and also please make sure that the subset check in splash.xml is also updated to "splash-screen"